particles/Renderer.cpp

```diff
@@ -277,8 +262,7 @@ void Renderer::init(score::gfx::RenderList& renderer)
             particlesUploaded = true;
         }
 
-        res.uploadStaticBuffer(particleControls, 0, sizeof(float), &n.particlesSpeedMod);
-        res.uploadStaticBuffer(particleControls, sizeof(float), sizeof(int), &n.particlesNumber);
```

Comment:
this second line was a bug that made it crash here: the problem is that the particleControls buffer was allocated with a size of sizeof(Controls) (e.g. 1 float). 
Thus we were writing out-of-bounds ; also this was not necessary at all since the particle count wasn't accessed anywhere, it's just needed as an isntance count.